### PR TITLE
feat : forgot password page

### PR DESCRIPTION
**Pull request by rohit8686**
Forgot Password Page - [Link](https://eleco.netlify.app/pages/forgotpassword/forgotpassword)

### CODE REVIEW

**Comment by Adarsh-Patel-Dev** [comment]
on mycart page give some space between remove from cart button and move to wishlist button
Also give same width to both buttons in left card(price checkout) 
Rest UI is clean

File: Pages/ForgotPassword/forgotPassword.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <link rel="stylesheet" href="forgotPassword.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined"
+    />
+    <title>Forgot Password</title>
```
Comment (theVivek-kumar) [inline Pages/ForgotPassword/forgotPassword.html:16]: gave a proper CSS naming convention BEM or ocss

File: Pages/ForgotPassword/forgotPassword.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  </head>
+  <body>
+    <header>
+      <nav class="nav flex space-between">
+        <a href="/index.html" class="link">
+          <h1 class="gradient-text title">Eleco</h1>
+        </a>
+        <div class="flex px-1">
```
Comment (theVivek-kumar) [inline Pages/ForgotPassword/forgotPassword.html:24]: gave meaningful name foe class

File: Pages/ForgotPassword/forgotPassword.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            <span class="material-icons span icon icon-size"> search </span>
+            <input
+              type="search"
+              name="search"
+              id="search"
+              class="search"
+              placeholder="Search"
+            />
+          </div>
+          <a href="/Pages/Login/login.html">
```
Comment (theVivek-kumar) [inline Pages/ForgotPassword/forgotPassword.html:35]: this  name is good

File: Pages/ForgotPassword/forgotPassword.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+              <span class="material-icons span icon icon-size">
+                favorite_border
+              </span>
+              <div class="number status flex busy">5</div>
+            </div>
+          </a>
+          <a href="/Pages/Cart/cart.html" class="link">
+            <div class="badge">
+              <span class="material-icons-outlined span icon icon-size">
+                shopping_cart
```
Comment (theVivek-kumar) [inline Pages/ForgotPassword/forgotPassword.html:49]: this looks good and clean

**Review by theVivek-kumar — APPROVED** [review]
can be merged

**Review by RahulSingh9131 — APPROVED** [review]
Code looks good to me, it can be merge.

**Pull request merged**